<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginDialogClass</class>
 <widget class="QDialog" name="CLoginDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLoginDialog</string>
  </property>
  <widget class="QFrame" name="formFrame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;烧豆腐&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="m_userLabel">
      <property name="text">
       <string>请输入用户名：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_passWdLabel">
      <property name="text">
       <string>请输入密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="m_passWdLineE"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="m_userLineE"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_pLoginBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登              录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pRegistBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注   册</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CLoginDialog.qrc"/>
 </resources>
 <connections/>
</ui>
